Makefile

```diff
@@ -18,3 +18,6 @@ deps:
 
 generate-easyjson:
 	docker run --rm -v ${PWD}:/src -w /src golang:1.20-alpine ./hack/generate-easyjson.sh
+
+generate-mocks:
+	docker run --rm -v ${PWD}:/src -w /src golang:1.20-alpine ./hack/generate-mocks.sh
```

Comment:
Check your local files. I believe that when we run this Makefile target, the `vendor` directory owner change to `root` user. At least, I can see that in my local environment. Maybe you need to set the user to be used inside the container.

Reply:
Fixed it.